iotedgedev/modules.py

```diff
@@ -27,86 +31,48 @@ def build(self):
 
                 self.output.info("BUILDING MODULE: {0}".format(module_dir))
 
-                # Find first proj file in module dir and use it.
-                project_files = [os.path.join(module_dir, f) for f in os.listdir(
-                    module_dir) if f.endswith("proj")]
+                module_json = Module(
```

Comment:
call this module instead of module_json

Reply:
that would conflict with the module var in for loop